hydromt_fiat/fiat.py

```diff
@@ -386,7 +394,7 @@ def setup_exposure_buildings(
                 asset_locations,
                 ground_floor_height,
                 extraction_method,
-                ground_elevation_file=ground_elevation_file,
+                ground_elevation=ground_elevation,
             )
```

Comment:
why do we not use the units in this method?

Reply:
its not used in the function, same as in the function below (took it out there)